nbs/methods/05_sphere.ipynb

```diff
@@ -229,19 +229,18 @@
     "        if configs is None:\n",
```

Comment:
<p>Can you check whether <code>GrowingSphere</code> generates different <code>cfs</code> or different <code>X</code>  ? Add a test case.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/ReLax/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>